src/core/network/createPayload.js

```diff
@@ -20,7 +20,8 @@ export default () => {
     addIdentity: (namespaceCode, identity) => {
       content.xdm = content.xdm || {};
       content.xdm.identityMap = content.xdm.identityMap || {};
-      content.xdm.identityMap[namespaceCode] = content.xdm.identityMap[namespaceCode] || [];
+      content.xdm.identityMap[namespaceCode] =
+        content.xdm.identityMap[namespaceCode] || [];
```

Comment:
For some reason people are able to commit+push when code doesn't match prettier rules. I have a note to figure out why this is happening.